src/pcre2_compile_class.c

```diff
@@ -2276,12 +2288,11 @@ return TRUE;
 /* This function consumes unary prefix operators. */
 
 static BOOL
-compile_class_unary(uint32_t options, uint32_t xoptions, BOOL negated,
+compile_class_unary(eclass_context *context, BOOL negated,
```

Comment:
This could be merged with compile_class_juxtaposition.

Reply:
It's clear as it is. I don't mind either way.